pgs_types.sql.in

```diff
@@ -132,6 +132,17 @@ CREATE FUNCTION spoly_out(spoly)
    LANGUAGE 'c'
    IMMUTABLE STRICT;
 
+CREATE FUNCTION spoly_deg(float8[])
+   RETURNS spoly
+   AS 'MODULE_PATHNAME', 'spherepoly_deg'
+   LANGUAGE 'c'
+   IMMUTABLE STRICT;
+
+COMMENT ON FUNCTION spoly_deg(float8[]) IS
+  'create spoly from array of points. '
+  'Two consecutive numbers among those present '
+  'refer to the same occurrence and cover its '
+  'latitude and longitude, respectively.';
```

Comment:
Does this (lines 143-145) work? The documentation says that "COMMENT ON <object> IS" must be followed by either a string literal or NULL. It doesn't say anything about multiple string literals?